doc/release-notes/rl-2511.section.md

```diff
@@ -59,6 +59,8 @@
 
 - `telegram-desktop` packages now uses `Telegram` for its binary. The previous name was `telegram-desktop`. This is due to [an upstream decision](https://github.com/telegramdesktop/tdesktop/commit/56ff5808a3d766f892bc3c3305afb106b629ef6f) to make the name consistent with other platforms.
 
+- `netbird-management` & `netbird-signal` packages got split out of `netbird` due to relicensing of server components to AGPLv3 with version `0.53.0`,
```

Comment:
Shouldn't this be in the NixOS changelog?

Reply:
Could you clarify? I'm not 100% sure what you mean.